editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -5890,22 +5910,109 @@ Ref<EditorSpatialGizmo> EditorSpatialGizmoPlugin::get_gizmo(Spatial *p_spatial)
 	return ref;
 }
 
+void EditorSpatialGizmoPlugin::_bind_methods() {
+#define GIZMO_REF PropertyInfo(Variant::OBJECT, "gizmo", PROPERTY_HINT_RESOURCE_TYPE, "EditorSpatialGizmo")
+
+	BIND_VMETHOD(MethodInfo(Variant::BOOL, "has_gizmo", PropertyInfo(Variant::OBJECT, "spatial", PROPERTY_HINT_RESOURCE_TYPE, "Spatial")));
+	BIND_VMETHOD(MethodInfo(GIZMO_REF, "has_gizmo", PropertyInfo(Variant::OBJECT, "spatial", PROPERTY_HINT_RESOURCE_TYPE, "Spatial")));
```

Comment:
Does it work to use a detailed `PropertyInfo` for the return type? Or is it the same as `Variant::OBJECT` in the end?

Reply:
Also is it expected to have two vmethods named `has_gizmo` with different return type?

Reply:
Looking at the code, I think it was meant to be `create_gizmo` here.

Reply:
To rule out potential issues, might be worth running `godot --doctool .` to generate the empty template, even if you'll only fill it later.